learning_resources/views.py

```diff
@@ -994,6 +1066,11 @@ def get_queryset(self) -> QuerySet:
             .distinct()
         )
 
+    @method_decorator(
```

Comment:
There was some deliberate results randomization added to this featured list view so as not to give any apparent preference to any particular unit.  Might be good to double-check with Ferdi to see if it's okay if that randomization is cached so all anonymous users see the same result for 24 hours before it's randomized again.

Reply:
PS and clear the cache after the news_events tasks or mgmt command are run.

Reply:
double checked with ferdi. He said we can leave this as-is and have a separate ticket to handle the randomization on the frontend.